web/gui/dashboard_info.js

```diff
@@ -835,6 +835,16 @@ netdataDashboard.submenu = {
     'apps.file_access': {
         title: 'File Access',
         info: 'Netdata also gives a summary for this chart on <a href="#menu_filesystem_submenu_File_access">Filesystem submenu</a> (more details on <a href="https://learn.netdata.cloud/docs/agent/collectors/ebpf.plugin#file" target="_blank">eBPF plugin file chart section</a>).'
+    },
+
+    'ip.kernel': {
+        title: 'Kernel Functions',
+        info: 'Next charts are made when <code>ebpf.plugin</code> is running on your host. When integration with apps is <a href="https://learn.netdata.cloud/guides/troubleshoot/monitor-debug-applications-ebpf" target="_blank">enabled</a>, Netdata also shows calls for kernel functions per <a href="#menu_apps_submenu_net">application</a>.'
+    },
+
+    'apps.net': {
+        title: 'Network monitoring',
```

Comment:
<img width="198" alt="Screenshot 2021-09-14 at 17 12 15" src="https://user-images.githubusercontent.com/22274335/133273804-8bdcc493-4199-4b74-b982-15ad81f919a6.png">

Somehow I missed that change.

- Why start with an uppercase letter? Do we change our current pattern? If so, why do we do it only for "net"?
- I doubt we need "monitoring" word, it adds no value (we _can_ add it to every menu section).

If there is a document you followed doing this change?

Reply:
You are right, it was typo error :/ . I will bring PR fixing all eBPF submenus.